modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -139,17 +139,22 @@ See xref:using-s3-for-blobs[Using S3 for Blobs] for the S3 configuration.
 
 === Configuration Directory
 
-The default locations (no need to set them explicitly) for _config_ files, which have to be on a POSIX storage, are:
+* The configuration directory has a default location for _config_ files, which *must be* on a POSIX storage:
 
-* For container images (inside the container) +
+** For container images (inside the container) +
 `/etc/ocis/`
 +
-* For binary releases +
-`$HOME/.ocis/config/` or `~/.ocis/config/` (preferred)
+** For binary releases +
+`$HOME/.ocis/config/`
 +
-NOTE: You can deviate from the default location and define a custom configuration file location on startup using the environment variable `OCIS_CONFIG_DIR`.
-+
-NOTE: When using a system user for the runtime, which has no login and therefore no home directory, like in the scenario xref:deployment/binary/binary-setup.adoc#setting-up-systemd-for-infinite-scale[Setting up systemd for Infinite Scale], you *must* specify a configuration file location.
+[NOTE]
+====
+* Do not replace `$HOME` with `~` (tilde). The code does not resolve `~` to the users home directory.
+* Check that `$HOME` resolves to a valid directory.
+** When using a system user for the runtime, which has no login and therefore no home directory, like in the scenario xref:deployment/binary/binary-setup.adoc#setting-up-systemd-for-infinite-scale[Setting up systemd for Infinite Scale], you *must* specify a configuration file location.
+====
+
+* You can deviate from the default location and define a custom configuration file location on startup using the environment variable `OCIS_CONFIG_DIR`.
```

Comment:
We should refer to 'configuration directory' everywhere.  157 should eventually read:
* You can deviate from the default location and define a custom configuration directory location on startup using the environment variable `OCIS_CONFIG_DIR`.

Reply:
done